kano_profile/badges.py

```diff
@@ -264,10 +345,15 @@ def save_app_state_with_dialog(app_name, data):
         return
 
     if is_gui():
-        with open(os.path.join(os.path.expanduser('~'), '.kano-notifications.fifo'), 'w') as fifo:
+        fifo = open(os.path.join(os.path.expanduser('~'),
+                                 '.kano-notifications.fifo'),
+                    'w')
```

Comment:
Maybe this line would be clearer as:

```python
fifo = open(
    os.path.join(os.path.expanduser('~'), '.kano-notifications.fifo'), 'w'
)
```

or, with the same number of lines as before:

```python
fifo = open(
    os.path.join(os.path.expanduser('~'), '.kano-notifications.fifo'),
    'w'
)
```
```